glusterd2/xlator/options/options.go

```diff
@@ -324,10 +324,84 @@ func ValidateStr(o *Option, val string) error {
 
 // ValidateTime validates if the option is valid time format
 func ValidateTime(o *Option, val string) error {
-	if validate.IsTime(val, "hh:mm:ss") != true {
```

Comment:
Is this condition not required in the end? (else part)

Reply:
the glusterd one the "xlator_option_validate_time" is the common validation function for time format and in that the "gf_string2time" doesn't have a parsing for "hh:mm:ss". 

We can add it if we want to suport it from GD2.